src/domain/events/eventSearchForm/eventSearchForm.module.scss

```diff
@@ -70,6 +70,17 @@
   }
 }
 
+.checkboxWrapper {
+    align-items: center;
```

Comment:
Seems like there's a 4 space indent here when the other rows in the file use 2.

Prettier should also support scss files.

Is this an issue with how the project has been configured considering that these haven't printed warnings?